services/datalad/datalad_service/common/git.py

```diff
@@ -40,8 +40,22 @@ def git_tag_tree(repo, tag):
     return repo.get(tag_reference.target).tree_id
 
 
+def git_rename_master_to_main(repo):
+    # Make sure the main branch is used, update if needed
+    master_branch = repo.branches.get('master')
+    if master_branch:
+        main_branch = master_branch.rename('main', True)
+        # Abort the commit if this didn't work
+        if not main_branch.is_head():
+            raise Exception('Unable to rename master branch to main')
+    # Always update the symbolic reference for HEAD
+    repo.references['HEAD'].set_target('refs/heads/main')
```

Comment:
What do you think of this?

```suggestion
    HEAD = repo.references["HEAD"]
    # Make sure the main branch is used, update if needed
    if HEAD.target == "refs/heads/master":
        main_branch = repo.branches.get('master').rename('main', True)
        # Abort the commit if this didn't work
        if not main_branch.is_head():
            raise Exception('Unable to rename master branch to main')
    # Always update the symbolic reference for HEAD
    HEAD.set_target('refs/heads/main')
    # Remove master if it's still around
    if "master" in repo.branches:
        repo.branches["master"].delete()
```